symposion/schedule/tests/factories.py

```diff
@@ -1,10 +1,44 @@
+import datetime
 import factory
+import factory.fuzzy
 
 from pycon.tests.factories import PyConTutorialProposalFactory
 from symposion.conference.tests.factories import SectionFactory
 from symposion.speakers.tests.factories import SpeakerFactory
 
-from ..models import Presentation
+from ..models import Presentation, Slot, SlotKind, Day, Schedule
+
+
+class ScheduleFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = Schedule
+
+    section = factory.SubFactory(SectionFactory)
+
+
+class DayFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = Day
+    schedule = factory.SubFactory(ScheduleFactory)
+    date = factory.fuzzy.FuzzyDate(start_date=datetime.date.today())
+
+
+class SlotKindFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = SlotKind
+
+    schedule = factory.SubFactory(ScheduleFactory)
+    label = factory.fuzzy.FuzzyText()
+
+
+class SlotFactory(factory.DjangoModelFactory):
+    FACTORY_FOR = Slot
```

Comment:
Do you think we should update all our factories?  Maybe that should be another pull request.